WordPress/WordPressShare/PostStatusPickerViewController.swift

```diff
@@ -0,0 +1,107 @@
+import Foundation
+import UIKit
+import WordPressShared
+
+class PostStatusPickerViewController : UITableViewController
+{
+    // MARK: - Initializers
+    init(statuses : [String: String]) {
+        // Note:  We'll store the sorted Post Statuses, into an array, as a (Key, Description) tuple.
+        self.sortedStatuses = statuses.sort { $0.1 < $1.1 }
+        super.init(style: .Plain)
+    }
+    
+    required init?(coder: NSCoder) {
+        sortedStatuses = [(String, String)]()
+        super.init(coder: coder)
+        assert(false, "Please, use the main initializer instead")
+    }
```

Comment:
Probably best to use a fatalError as Xcode suggests

```
    required init?(coder aDecoder: NSCoder) {
        fatalError("init(coder:) has not been implemented")
    }
```

Also, FYI, `assert(false, ...)` == `assertionFailure(...)`